src/app/layout.tsx

```diff
@@ -1,8 +1,9 @@
-import Footer from '@/components/footer/footer';
-import Header from '@/components/header/header';
 import type { Metadata } from 'next';
 import { Inter } from 'next/font/google';
-import './globals.css';
+import './globals.scss';
```

Comment:
Did this auto format?

Reply:
I changed it. Should I change it back to CSS?